### PR TITLE
shark party

### PR DESCRIPTION
**Pull request by aaronj1335**
cc @lawnsea @andycouch

@tswicegood and @timtyrrell thank you gentlemen for being willing to speak!

@qouyang let me know if there's a specific link you'd like to use.

I'll post this tomorrow a bit before lunch if i don't hear anything.

### CODE REVIEW

File: _posts/2015-09-15-meetup.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+---
+
+Webster's Dictionary [defines "extravaganza" as][extravaganza]:
+
+> a very large and exciting show or event
+
+And nothing says exciting like JavaScript frameworksssssss!
+
+<div class="sponsor-logo">
+  <img src="https://cldup.com/D3VCZfiOrY.gif" style="width: 300px; max-width: 100%;" alt="WHOA REACT JS" />
```
Comment (lawnsea) [inline _posts/2015-09-15-meetup.md]: Is that image [safe for those with seizure disorders](http://webaim.org/articles/seizure/)?

**Comment by lawnsea** [comment]
LGTM other than my a11y concern

File: _posts/2015-09-15-meetup.md (hunk 1 above)
Comment (aaronj1335) [inline _posts/2015-09-15-meetup.md]: whoa good point. would one of these work? we need something safe, but i'd still like it to be as offensive to the eyes as possible:

hypno:

hypno

miami night club:

night club

drunk goggles:

drunk goggles

Comment (aaronj1335) [inline _posts/2015-09-15-meetup.md]: or maybe a still of the night club:

night club freeze

Comment (lawnsea) [inline _posts/2015-09-15-meetup.md]: the animated miami night club seems ok to me, but i'm not an expert.

we could do an acrostic in comic sans instead:
Really
Excellent
Awesome
Cool
Tubular
Jammin
Spectacular

Comment (aaronj1335) [inline _posts/2015-09-15-meetup.md]: how about i do the animated one with a big button that says "don't ever show that to me again"?

**Comment by aaronj1335** [comment]
updated

**Pull request closed without merging**